site/deploy.md

```diff
@@ -28,6 +28,10 @@ When running the tool in WLST online mode, the deploy operation may require serv
 - `103` - The entire domain needs to be restarted.
 - `104` - The domain changes have been rolled back because the changes in the model requires a domain restart and -rollback_if_require_restart is specified.
 
+### Using an Encrypted Model
+
+If the model or variables file contains encrypted passwords, add the `-use_encryption` flag to the command line to tell the Create Domain Tool that encryption is being used and to prompt for the encryption passphrase.  As with the database passwords, the tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input.
```

Comment:
Any encrypted passwords?  Encrypted with domain SSID too?  I think you should be more specific with "...file contains encrypted passwords..." to avoid confusion and questions.  And, the "as with database passwords..." is confusing to me.  I think I would drop everything before the comma and go with "The tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input"

Reply:
"If the model or variables file was encrypted with the Encrypt Model Tool, ...." ?